src/Lime.Protocol.UnitTests/Serialization/TypeUtilTests.cs

```diff
@@ -51,5 +51,17 @@ public void GetFormatterParseFunc_DoubleSpecificCulture_ReturnValidParser()
             // Assert
             result.ShouldBe(100_000);
         }
+
+        [Test]
+        public void GetStringValue_EmptyStringToEmptyArray_ReturnEmptyArray()
+        {
+            object result = null;
+
+            //Act
+            TypeUtilEx.TryParseString("", typeof(string[]), out result, CultureInfo.InvariantCulture);
+
+            result.ShouldBeOfType(typeof(string[]));
```

Comment:
`ShouldBeOfType` returns an instance of the requested type (`string[]`) if the assertion passes. You can use this to remove the cast on next line.